grails-app/domain/org/bbop/apollo/FeatureLocation.groovy

```diff
@@ -16,19 +17,19 @@ class FeatureLocation {
         residueInfo nullable: true
         locgroup nullable: true
         rank nullable: true
-
     }
 
-    Feature feature;
-    Integer fmin;
-    boolean isFminPartial;
-    Integer fmax;
-    boolean isFmaxPartial;
-    Integer strand;
-    Integer phase;
-    String residueInfo;
-    int locgroup;
-    int rank;
+    Feature feature
+    Integer fmin
+    Integer length
```

Comment:
I tried transient and it did not work. I am using the recommended syntax for a mapped property so I think instantiating it this was is fine http://grails.github.io/grails-doc/2.4.5/guide/GORM.html#derivedProperties